guide-us-english.tex

```diff
@@ -276,13 +276,17 @@ \section{Scrum Master Cycle - Coordinating the ``How''}
 
 The SM organization (SM's, SoSM's, and EAT) work as a whole to complete the components of the Scrum Master Cycle. The components unique to the Scrum Master Cycle are: 
 \begin{itemize}
-	\item \textbf{Continuous Improvement and Impediment Removal,}
-	\item \textbf{Cross-Team Coordination, and}
-	\item \textbf{Deployment.}
+	\item \textbf{Continuous Improvement and Impediment Removal},
+	\item \textbf{Cross-Team Coordination}, and
+	\item \textbf{Deployment}.
 \end{itemize}
 
 \subsection{Continuous Improvement and Impediment Removal}
 
+In order to scale, impediments need to be removed as soon as possible.
+Unresolved impediments scale with the implementation and cripple productivity.
+In Scrum@Scale we crush impediments to guarantee a scale-free architecture.
+
```

Comment:
"In Scrum@Scale we crush impediments"... don't like this part. Too propaganda-ish. "We will CRUSH our enemies!!"